### DISCUSSION TITLE
Use existing SD models?

### DISCUSSION THREAD

**Discussion by gerroon** [tin2tin/Pallaidium#80] [Q&A] [answered]
Hi

I already have AUTOMATIC1111's SD install and I have the .safetensors installed there. Is there a way to use those models nstead of downloading them over ?

Thanks

**Comment by tin2tin** [comment, marked as answer]
Diffusers downloads and installs them in a .cache folder. You can try to download one model and see the folder structure, you'll need to imitate for the other files. I'm not using automatic1111, so I do not know the specifics.
